web/public/locales/sv.json

```diff
@@ -1,4 +1,18 @@
 {
+  "info-modal": {
+    "title": "Info",
+    "intro-text": "Appen electricityMap ger insikter i realtid på webben och mobil för medborgare, energiexperter, icke-statliga organisationer och beslutsfattare och främjar en djupare förståelse för utmaningar i energiomställningen.",
+    "open-source-text": "Den här appen är <a href=\"%s\" target=\"_blank\">Open Source</a> (<a href=\"%s\" target=\"_blank\">se källkoden</a>) och drivs av en community.",
```

Comment:
```suggestion
    "open-source-text": "Den här appen är <a href=\"%s\" target=\"_blank\">Open Source</a> (<a href=\"%s\" target=\"_blank\">se källkoden</a>) och drivs av en gemenskap.",
```

Reply:
I agree it sounds a little weird but since we use "Slack-gemenskapen" for slack I think it makese sense.